plugins/scaffolder/src/next/OngoingTask/ContextMenu.tsx

```diff
@@ -90,6 +100,14 @@ export const ContextMenu = (props: ContextMenuProps) => {
             </ListItemIcon>
             <ListItemText primary={logsVisible ? 'Hide Logs' : 'Show Logs'} />
           </MenuItem>
+          <MenuItem onClick={() => onToggleButtonBar?.(!buttonBarVisible)}>
+            <ListItemIcon>
+              <ControlPointIcon fontSize="small" />
+            </ListItemIcon>
+            <ListItemText
+              primary={buttonBarVisible ? 'Hide Button Bar' : 'Show Button Bar'}
+            />
+          </MenuItem>
```

Comment:
This one I don't quite understand - is it really desirable to toggle this bar at all? Seems like it should be safe to be opinionated here and having it visible and always useful.

Reply:
So this was my attempt at a compromise between not having the button bar at all which is the current state and having the button bar. Right now it's only not shown when everything is successful. Would it make sense to just remove the context menu option but keep the way it's shown in this PR?

What do you think @benjdlambert?

Reply:
I, for one, would vote to always have it!

Reply:
I'm happy to merge this as is to get it in the release for today - but @ahhhndre if you want to raise a PR to have it at all times that would be great.